bin/test.sh

```diff
@@ -57,6 +57,8 @@ done
 : "${PHPUNIT_VERSION:=""}"
 : "${DOCKER_OPTIONS:=""}"
 
+PHP_OPTIONS="-d apc.enable_cli=1 ${PHP_OPTIONS}"
```

Comment:
APCu is not enabled in the CLI mode by default. We cannot move this to `phpunit.xml.dist` because `ap.enable_cli` is `PHP_INI_SYSTEM`.

Reply:
What does this do?

Reply:
It prepends `-d apc.enable_cli=1` to the `PHP_OPTIONS` variable. It is then [passed to Docker](https://github.com/Automattic/vip-go-mu-plugins/blob/57636750224f1c3732076f07b29601dd4226f41d/bin/test.sh#L116). The entrypoint script [passes these options](https://github.com/Automattic/vip-container-images/blob/master/wp-test-runner/runner.sh#L13) to PHP.

We will end up with

```bash
php -d apc.enable_cli=1 other parameters go here
```

The `-d` option [defines a php.ini value](https://www.php.net/manual/en/features.commandline.options.php).

In this case, we [enable APCu in the CLI](https://www.php.net/manual/en/apcu.configuration.php#ini.apcu.enable-cli).